### PR TITLE
Doc:release_notes_2.1 update

### PR DESCRIPTION
**Pull request by fuzhongl**
Add fixed issues and known issues.

Signed-off-by: fuzhongl <fuzhong.liu@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/release_notes/release_notes_2.1.rst [hunk 1]
```diff
@@ -36,7 +36,7 @@ We recommend that all developers upgrade to ACRN release v2.1.
 What's new in v2.1
 ******************
 
-* Preempt-RT Linux has been validated as a pre-launched realtime VM.
+* Preempt-RT Linux has been validated as a pre-launched realtime VM.See :ref:`pre_launched_rt`
```
Comment (dbkinder) [inline doc/release_notes/release_notes_2.1.rst:39]: needs a space after the end of sentence period

File: doc/release_notes/release_notes_2.1.rst [hunk 2]
```diff
@@ -67,9 +67,40 @@ including these:
 
 Fixed Issues Details
 ********************
-
-.. comment Issues should be carefully formatted like this
-   - :acrn-issue:`3715` -  Add support for multiple RDT resource allocation and fix L3 CAT config overwrite by L2
+- :acrn-issue: `4047` -  [WHL][Function][WaaG] passthru usb, Windows will hang when reboot it
```
Comment (dbkinder) [inline doc/release_notes/release_notes_2.1.rst:70]: There must not be a space after the colon in ``:acrn-issue:`4047` ``

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/release_notes/release_notes_2.1.rst (hunk 2 above)
Comment (fuzhongl) [reply doc/release_notes/release_notes_2.1.rst]: Will take care this issue next time.
Thanks.

**Pull request merged**